Error handling for non-existent file paths

Get the following error when attempting to do a diff on a non-existent file path:

```bash
goff argocd app "./foo/bar" --repo-server="localhost:8081" --output-dir=/tmp/source/

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x28d72be]

goroutine 1 [running]:
github.com/puzzle/goff/argocd.findArgoApps.func1({0x7ff7bfefeb70, 0x1e}, {0x0, 0x0}, {0xc000a9af30?, 0xc000a19a00?})
        /src/argocd/repoClient.go:167 +0x3e
path/filepath.Walk({0x7ff7bfefeb70, 0x1e}, 0xc000a19b60)
        /usr/local/go/src/path/filepath/path.go:569 +0x50
github.com/puzzle/goff/argocd.findArgoApps({0x7ff7bfefeb70?, 0x0?})
        /src/argocd/repoClient.go:166 +0x65
github.com/puzzle/goff/argocd.Render({0x7ff7bfefeb70, 0x1e}, {0x7ff7bfefeb9c, 0xe}, {0x7ff7bfefebb8, 0xc}, {{0x0, 0x0}, {0x0, 0x0}, ...})
        /src/argocd/repoClient.go:37 +0x135
github.com/puzzle/goff/cmd/argocd.glob..func1(0x4279660?, {0xc000a91ad0?, 0x3?, 0x3?})
        /src/cmd/argocd/app.go:21 +0xde
github.com/spf13/cobra.(*Command).execute(0x4279660, {0xc000a91a70, 0x3, 0x3})
        /go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:944 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x4278dc0)
        /go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/puzzle/goff/cmd.Execute()
        /src/cmd/root.go:33 +0x45
main.main()
        /src/main.go:11 +0x17
```

--- a/argocd/appSet.go
+++ b/argocd/appSet.go
@@ -21,6 +21,11 @@ func RenderApplicationSets(inputDir, outDir string) error {
 
 	files := make([]string, 0)
 	filepath.WalkDir(inputDir, func(path string, d fs.DirEntry, err error) error {
+
+		if err != nil {
+			return err
+		}
+
 		if !d.IsDir() {
 			files = append(files, path)
 		}

--- a/argocd/repoClient.go
+++ b/argocd/repoClient.go
@@ -160,6 +160,11 @@ func renderFile(file, repoServerUrl, outputDir string, client apiclient.RepoServ
 func findArgoApps(rootDir string) ([]string, error) {
 	var argoAppFiles []string
 	err := filepath.Walk(rootDir, func(path string, info fs.FileInfo, err error) error {
+
+		if err != nil {
+			return err
+		}
+
 		if strings.HasSuffix(info.Name(), ".yml") || strings.HasSuffix(info.Name(), ".yaml") {
 
 			data, err := os.ReadFile(path)

--- a/kustomize/kustomizationfile/kustomizationfile.go
+++ b/kustomize/kustomizationfile/kustomizationfile.go
@@ -59,6 +59,7 @@ func (k *kustomizationFileContext) GetDirectories(directoryRootPath string) ([]s
 
 	directories := make([]string, 0)
 	err = filepath.Walk(directoryRootPath, func(path string, info os.FileInfo, err error) error {
+
 		if err != nil {
 			return err
 		}

